lib/api/core/entrypoints/embedded/index.js

```diff
@@ -358,20 +381,28 @@ class EmbeddedEntryPoint extends EntryPoint {
         url += '?' + queryString;
       }
 
-      if (request.input.jwt) {
+      if (user === null && request.input.jwt) {
         try {
           const
             b64Payload = request.input.jwt.split('.')[1],
             payload = Buffer.from(b64Payload, 'base64').toString('utf8');
           user = JSON.parse(payload)._id;
         }
         catch (err) {
-          this.kuzzle.pluginsManager.trigger('log:warn', `Unable to extract user from jwt token: ${request.input.jwt}`);
+          // do nothing: we don't know anything about the token, it may be
+          // invalid on-purpose for all we know
```

Comment:
I don't understand this comment : how and in wich case the token would be invalid on-purpose ?

Reply:
Forgery.

We're dealing with data coming from the outside, so nothing can be trusted.